std-bits/table/src/main/java/org/enso/table/data/table/Column.java

```diff
@@ -120,12 +122,27 @@ public static Column fromItems(String name, List<Value> items) {
    * @return a column with given name and items
    */
   public static Column fromRepeatedItems(String name, List<Value> items, int repeat) {
+    if (repeat < 1) {
+      throw new IllegalArgumentException("Repeat count must be positive.");
+    }
+
+    if (repeat == 1) {
+      return fromItems(name, items);
+    }
+
     var totalSize = items.size() * repeat;
+
+    // ToDo: This a workaround for an issue with polyglot layer. #5590 is related.
```

Comment:
So cool that now issue numbers work flawlessly in code preview on GH 😁

Reply:
It is unclear to me how to "turn off" the mentioned workaround, if I were to try reproducing this bug.

Maybe we can add a variable here `boolean workaround=true;`?